docs/application/tizen-studio/index.md

```diff
@@ -9,7 +9,7 @@ In the Tizen Studio, you can perform all the same development activities that we
 
 > **Note**
 > 
-> From Tizen Studio 3.1 onwards, support for 32 bit host for Windows and Ubuntu will be removed.
+> From Tizen Studio 3.5 onwards, support for 32 bit host for Windows and Ubuntu will be removed.
```

Comment:
A reader can be confused whether Tizen Studio 3.5 is release or not if you specify version number.

How about the following sentence?
The next version of Tizen Studio won't support 32 bit host for Windows and Ubuntu.

Reply:
Updated the sentence as recommended.